pkg/services/ngalert/api/api.go

```diff
@@ -87,4 +90,41 @@ func (api *API) RegisterAPIEndpoints(m *metrics.Metrics) {
 		DatasourceCache: api.DatasourceCache,
 		log:             logger,
 	}, m)
+	api.RegisterAdminAPIEndpoints(AdminSrv{
+		store: api.AdminConfigStore,
+		log:   logger,
+	}, m)
+}
+
+func (api *API) RegisterAdminAPIEndpoints(srv AdminSrv, m *metrics.Metrics) {
+	api.RouteRegister.Group("", func(group routing.RouteRegister) {
+		group.Get(
+			toMacaronPath("/api/v1/ngalert/configuration"),
```

Comment:
It was not obvious to me from the route what configuration it was pointing to until I looked through the code for the struct. How about `/api/v1/ngalert/admin_config` or something which tells what config it is?

Reply:
> It's not clear to me either. How we can tell that this endpoint is for getting external Alertmanagers?

That's deliberate - there's more to add over time here that's not necessarily external alertmanagers e.g. a setting to stop execution like the one we have in https://github.com/grafana/grafana/pull/35623